src/Components/header.jsx

```diff
@@ -0,0 +1,8 @@
+function Header() {
+    return (
+    <section>
```

Comment:
If this is a header, consider using the `header` semantic tag rather than `section`